### PR TITLE
Adding proper guides section

### PR DESCRIPTION
**Pull request by tapaswenipathak**
Fix https://github.com/ipfs/docs/issues/215.

### CODE REVIEW

File: content/guides/_index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: IPFS Guides
+headless: true
+opened: true
+---
+
+# [Install IPFS](https://docs.ipfs.io/guides/install/)
+
+Install page describes the variety of ways you can have IPFS on the machine.
```
Comment (jessicaschilling) [inline content/guides/_index.md]: _@tapaswenipathak, I suggest changing this section to the following for clarity:_

Instructions for installing a copy of IPFS on your system via your choice of the following options:

- Installing from a Prebuilt Package (recommended)
- Installing from ipfs-update
- Building from source

Also includes information on how to upgrade IPFS, and general troubleshooting instructions.

File: content/guides/_index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- Installing from ipfs-update
+- Building from source
+- Upgrading IPFS
+- Troubleshooting
+
+You should be using command line for these instructions.
+
+# [Transferring a File](https://github.com/ipfs/go-ipfs/blob/master/docs/file-transfer.md)
+
+# [Repliacting Large Datasets](https://github.com/ipfs/archives/tree/master/tutorials/replicating-large-datasets)
```
Comment (jessicaschilling) [inline content/guides/_index.md]: "Repliacting" should be "Replicating"

File: content/guides/_index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
```
Comment (jessicaschilling) [inline content/guides/_index.md]: Is this the right header syntax here? @olizilla or @momack2 or @cwaring or @meiqimichelle -- is this something on which you can advise? Thank you!

**Comment by ericronne** [comment]
Whoops looks like we were working in parallel @jessicaschilling 👻 

My tack was to simplify the overall page as an index, while unpacking each referenced sub-topic. I'll simply paste here, as opposed to making in-line suggestions or committing a new page. Feel free to take or leave any of the content …

# [Install IPFS](https://docs.ipfs.io/guides/install/)
Learn the various ways to get IPFS up and running locally on your machine, and debug problems

# [Transfer a File](https://github.com/ipfs/go-ipfs/blob/master/docs/file-transfer.md)
A guide to help troubleshoot file transfers between two machines

# [Replicate Large Datasets](https://github.com/ipfs/archives/tree/master/tutorials/replicating-large-datasets)
Find out how to efficiently copy and provide (or mirror) large amounts of data

# [Address IPFS in Browsers](https://github.com/ipfs/in-web-browsers/blob/master/ADDRESSING.md)
An overview of methods for browser-based content access

# [Interactive Tutorials](https://proto.school/#/tutorials)
Self-guided instruction modules, with live code challenges

File: content/guides/_index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+- Installing from a Prebuilt Package
+- Installing from ipfs-update
+- Building from source
+- Upgrading IPFS
+- Troubleshooting
+
+You should be using command line for these instructions.
+
+# [Transferring a File](https://github.com/ipfs/go-ipfs/blob/master/docs/file-transfer.md)
+
```
Comment (jessicaschilling) [inline content/guides/_index.md]: _Please add as in @ericronne comment:_
A guide to help troubleshoot file transfers between two machines

File: content/guides/_index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+# [Repliacting Large Datasets](https://github.com/ipfs/archives/tree/master/tutorials/replicating-large-datasets)
+
```
Comment (jessicaschilling) [inline content/guides/_index.md]: _Please add as in @ericronne comment:_
Find out how to efficiently copy and provide (or mirror) large amounts of data

File: content/guides/_index.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+# [Replicating Large Datasets](https://github.com/ipfs/archives/tree/master/tutorials/replicating-large-datasets)
```
Comment (jessicaschilling) [inline content/guides/_index.md]: Remove -- this is redundant

File: content/guides/_index.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+# [Addressing in Browsers](https://github.com/ipfs/in-web-browsers/blob/master/ADDRESSING.md)
+
```
Comment (jessicaschilling) [inline content/guides/_index.md]: _Please add as in @ericronne comment:_
An overview of methods for browser-based content access

File: content/guides/_index.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+# [Interactive Tutorials](https://proto.school/#/tutorials)
```
Comment (jessicaschilling) [inline content/guides/_index.md]: _Please add as in @ericronne comment:_
Self-guided instruction modules, with live code challenges

**Comment by jessicaschilling** [comment]
@tapaswenipathak -- if you could kindly incorporate @ericronne and my comments, that would be great. Thank you!

**Comment by tapaswenipathak** [comment]
@jessicaschilling @ericronne: I just addressed review comments, pl. let me know if there is anything else.

**Comment by jessicaschilling** [comment]
Made a few minor copy edits for consistency; removed text recommending command-line, since it's too detailed for the scope of this document; retained the double "guides" in file path, so as not to break menu structure (and we will be addressing this in overall docs platform revamp anyway soon). Merging, closes #215. Thanks!

**Pull request merged**